DataFormats/MuonReco/src/Muon.cc

```diff
@@ -147,6 +166,23 @@ unsigned int Muon::stationMask( ArbitrationType type ) const
          continue;
       }
 
+      if(type == ME0HitAndTrackArbitration) {
+      	 if(chamberMatch->me0Matches.empty()) continue;
+
+
+         for( std::vector<MuonSegmentMatch>::const_iterator me0Match = chamberMatch->me0Matches.begin();
+      	      me0Match != chamberMatch->me0Matches.end(); me0Match++ )
+      	   {
+      	     //Not sure if mask is correct..
+      	     curMask = 1<<( (chamberMatch->station()-1)+4*(chamberMatch->detector()-1) );
```

Comment:
There is only one station (1), and the detector is always 5, if it's actually ME0.
So, the result is (0 + 16). curMask should correspond to a sensible detector layer that can be assigned to the place where the segments are.
In the main thread I suggested 5.